### PR TITLE
Add [return: NotNullIfNotNull("str")] to SecurityElement.Escape

### PR DESCRIPTION
**Pull request by Bond-009**
This only returns null if str is null

### CODE REVIEW

**Comment by GrabYourPitchforks** [comment]
Thanks for the submission. Don't forget to update the .cs file in the ref folder as well! Otherwise callers won't see the new annotation.

File: src/libraries/System.Runtime/ref/System.Runtime.cs [hunk 1]
```diff
@@ -10508,6 +10508,7 @@ public sealed partial class SecurityElement
         public string? Attribute(string name) { throw null; }
         public System.Security.SecurityElement Copy() { throw null; }
         public bool Equal([System.Diagnostics.CodeAnalysis.NotNullWhenAttribute(true)] System.Security.SecurityElement? other) { throw null; }
+        [return: NotNullIfNotNull("str")]
```
Comment (GrabYourPitchforks) [inline src/libraries/System.Runtime/ref/System.Runtime.cs:10511]: In ref assemblies, attributes must be fully qualified: `[return: System.Diagnostics.BlaBlah...()]`

File: src/libraries/System.Runtime/ref/System.Runtime.cs [hunk 2]
```diff
@@ -10508,6 +10508,7 @@ public sealed partial class SecurityElement
         public string? Attribute(string name) { throw null; }
         public System.Security.SecurityElement Copy() { throw null; }
         public bool Equal([System.Diagnostics.CodeAnalysis.NotNullWhenAttribute(true)] System.Security.SecurityElement? other) { throw null; }
+        [return: System.Diagnostics.CodeAnalysis.NotNullIfNotNull("str")]
```
Comment (GrabYourPitchforks) [inline src/libraries/System.Runtime/ref/System.Runtime.cs:10511]:
```suggestion
        [return: System.Diagnostics.CodeAnalysis.NotNullIfNotNullAttribute("str")]
```

(see example on line 10510 above or line 10514 below: we tend to use the full type name, including namespace _and Attribute suffix_)

**Review by GrabYourPitchforks — CHANGES_REQUESTED** [review]

File: src/libraries/System.Runtime/ref/System.Runtime.cs (hunk 2 above)
Comment (Bond-009) [reply src/libraries/System.Runtime/ref/System.Runtime.cs]: Oops, third time's the charm

**Review by GrabYourPitchforks — APPROVED** [review]
Thanks! :)

**Pull request merged**